emukit/quadrature/kernels/integration_measures.py

```diff
@@ -42,8 +44,28 @@ def get_box(self) -> List[Tuple[float, float]]:
         """
         raise NotImplementedError
 
+    @property
+    def can_sample(self):
+        """True if combined with IntegrationMeasureSampler"""
```

Comment:
not sure what 'combined with IntegrationMeasureSampler' means here, can you elaborate?

Reply:
IntegrationMeasureSampler is the class defined below. `can_sample` is sth like `has_gradients`, so there may or may not be sampling capabilities.

Now that I think of it I can also just add the method `get_samples`to IntegrationMeasure and not implement it if we cannot sample. Would that be better? That is how it is handled in the Acquisition class currently.

Reply:
it's fine to stick to approach similar to acqusition, with can_sample and get_samples

Reply:
done